design/sketch.webidl

```diff
@@ -469,7 +470,7 @@ dictionary GPURenderPipelineDescriptor : GPUPipelineDescriptorBase {
     GPURasterizationStateDescriptor rasterizationState;
     sequence<GPUBlendStateDescriptor> blendStates;
     GPUDepthStencilStateDescriptor depthStencilState;
-    GPUInputStateDescriptor inputState;
+    GPUVertexBufferStateDescriptor vertexBufferState;
```

Comment:
small alternative: GPUVertexInputStateDescriptor vertexInputState

Reply:
To bikeshed, "input state" better describes what this is since it also contains the index format.